src/Track/index.tsx

```diff
@@ -133,20 +134,42 @@ export const Track: React.FC<Props> = ({ id, onRemove, metronome }) => {
         }
 
         if (event.data.message === 'SHARE_RECORDING_BUFFER') {
-          const recordingLength = event.data.recordingLength
+          const fullRecordingLength = event.data.recordingLength
+          // When in doubt... use dimensional analysis! 🙃
+          //
+          //  60 seconds    beats       60 seconds    minute
+          // ———————————— ➗ —————   🟰  ——————————— 𝒙  ———————      =>
+          //   minute      minute        minute       beats
+          //
+          //   seconds    minutes   measures    beats     samples     samples
+          //  ————————— 𝒙 ———————— 𝒙 ———————— 𝒙 ———————— 𝒙 ————————— 🟰 ——————————
+          //   minute     beat      loop       measure    second       loop
```

Comment:
for some reason this doesn't align super well in my editor compared to github... seems like a weird font issue